pkgs/top-level/all-packages.nix

```diff
@@ -9761,6 +9761,10 @@ in
 
   leaps = callPackage ../development/tools/leaps { };
 
+  ### DEVELOPMENT / JAVA MODULES
+
+  javaPackages = recurseIntoAttrs (callPackage ./java-packages.nix { });
```

Comment:
That's going to mess with my lists. Is there a straight-forward `List -> Set` function?
`callPackages` wants things to be sets.